server/routes/cassandra/tittle.js

```diff
@@ -0,0 +1,82 @@
+const router = require('express').Router();
+const Promise = require('bluebird');
+const axios = require('axios');
+const cassandra = require('cassandra-driver');
+let seedUri = 'http://localhost:3001/api/cassandra/scaled-seed/';
+const client = new cassandra.Client({
+  contactPoints: ['127.0.0.1'],
+  localDataCenter: 'datacenter1',
+  keyspace: 'tittle'
+});
+const generateCassanInsertQueries = require('../../example.data').generateCassanInsertQueries
+
+
+
+let writeScaledTittles = (currentCounts, cb) => {
+  let tittleQueries = generateCassanInsertQueries(currentCounts);
+  console.log('writting tittles to db...');
+
+  client.batch(tittleQueries, { prepare: true })
+    .then(() => {
+      let newCounts = Number(currentCounts) + 1400;
+      cb(null, newCounts);
+    })
+    .catch((err) => cb(`SEEDING TITTLES ERROR = ${err}`, null))
+};
+writeScaledTittles = Promise.promisify(writeScaledTittles);
```

Comment:
This writeScaledTitles could be refactored to start out returning a new promise as opposed to creating a function then promisifying it.

Reply:
return new Promise with resolve & reject?